src/utils.rs

```diff
@@ -115,4 +134,40 @@ mod tests {
             assert_line_written("foo bar\tbaz", true, 8, "foo bar baz");
         }
     }
+
+    mod modification_time {
+        use super::*;
+
+        #[test]
+        fn set_time() {
+            use chrono::{DateTime, Local};
+            use std::fs::File;
+            use std::time::SystemTime;
+
+            let target = "target/utils";
+            let _ = std::fs::create_dir(target);
+            let filename = &format!("{target}/foo");
+            let temp = File::create(filename).unwrap();
+
+            // set file modification time equal to current time
+            let current = SystemTime::now();
+            let _ = temp.set_modified(current);
+
+            // format current time
+            let current: DateTime<Local> = current.into();
+            let current: String = current.format("%Y-%m-%d %H:%M:%S%.9f %z").to_string();
+
+            // verify
+            assert_eq!(current, get_modification_time(filename));
+        }
+
+        #[test]
+        fn invalid_file() {
+            let invalid_file = "target/utils/invalid-file";
+
+            let m_time = get_modification_time(invalid_file);
+
+            assert!(!m_time.is_empty());
```

Comment:
How about storing the value of `SystemTime::now()` in a variable before calling `get_modification_time(invalid_file)`, and then asserting that `m_time` is greater than the value of that variable?

Reply:
Makes sense!